projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -76,22 +76,33 @@ async function getMilestoneDates( plugin, nextMilestone ) {
 		// Spaces between words.
 		.join( ' ' );
 
+	let pluginMessage = '';
+	if ( ! releaseDate ) {
+		pluginMessage = `No scheduled milestone found for this plugin.`;
+	} else if ( plugin === 'jetpack' ) {
+		pluginMessage = `The Jetpack plugin has different release cadences depending on the platform:
+
+- WordPress.com Simple releases happen semi-continuously (PCYsg-Jjm-p2).
```

Comment:
Would that be an opportunity to tell people they're expected to stick around to deploy to simple?

```suggestion
- WordPress.com Simple releases happen as soon as you deploy your changes after merging this PR (PCYsg-Jjm-p2).
```